.circleci/river.env

```diff
@@ -2,7 +2,7 @@ export ZOWE_CLI_PROFILE_NAME="river"
 export TEST_PROFILE="zos"
 export ZOS_HOST="river.zowe.org"
 export ZOS_ZOSMF_PORT="10443"
-export ZOS_SSH_PORT="2022"
```

Comment:
Why this change of River's configuration? Is it possible to use it in the future (or it is better to delete)?

Reply:
I think that we should delete it. My understanding is that River is dead and cannot be resurrected.

Reply:
You still can connect to USS using SSH on River. However, zos configuration is broken. I still have seen some ideas to restore River configuration from backup. But they were a little bit vague.